test/snapshot/call.test.js

```diff
@@ -253,6 +253,15 @@ describe("Test call", function () {
     });
     expect(astErr).toMatchSnapshot();
   });
+  it("doesnt confused static methods with named arguments", function () {
+    const astErr = parser.parseEval(`foo(a::bar());`, {
+      parser: {
+        version: "8.0",
```

Comment:
should be interesting to see behavior with `version: "7.2"`

Reply:
Good point! I'll send a PR with a test.

Reply:
done! I implemented the version check and added a test.